src/Common/Geocoder.php

```diff
@@ -45,12 +45,12 @@ public function geocode(string $value): Collection;
     /**
      * Reverses geocode given latitude and longitude values.
      *
-     * @param float $latitude
-     * @param float $longitude
+     * @param float|null $latitude
+     * @param float|null $longitude
      *
      * @return Collection
      *
-     * @throws \Geocoder\Exception\Exception
+     * @throws Exception\Exception
      */
-    public function reverse(float $latitude, float $longitude): Collection;
+    public function reverse(?float $latitude = null, ?float $longitude = null): Collection;
```

Comment:
This is a BC break

Reply:
Is the whole PR a BC break or just this bit of code?

Reply:
Interface change :)

Reply:
I have changed the interface now and all of its implementations